clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/ServerGroupController.groovy

```diff
@@ -76,6 +79,30 @@ class ServerGroupController {
     serverGroup
   }
 
+  @RequestMapping(value = "/serverGroupsDetailsForList", method = RequestMethod.POST)
+  List<ServerGroup> getServerGroupsDetailsForList(@RequestBody String serverGroupList) {
+
+    List<ServerGroup> results = [];
+    serverGroupList.split(',').collect {
```

Comment:
I feel like there's a better way to create the map from the pipe-delimited values, but I played around with `inject` far too long and wasn't terribly successful.